tin/apps/assignments/forms.py

```diff
@@ -55,6 +64,31 @@ class Meta:
             "submission_limit_cooldown": "Rate limit cooldown period (minutes)",
             "is_quiz": "Is this a quiz?",
         }
+        sections = {
+            "Required": (
+                "name",
+                "description",
+                "due",
+                "points_possible",
+                "is_quiz",
+                "hidden",
+            ),
+            "Environment Setup": (
+                "folder",
+                "language",
+                "filename",
```

Comment:
I don't think "folder," "language," and "filename" should be collapsed by default - they're very important and can't be left to default values. A few ways to fix this:
- Restructure this dictionary to support section attributes,. What about  
`[{"name": "Environment Setup", "description": "", "fields": [...], "collapsed": False}, ...]`?
- Move these fields into the main section